.github/workflows/on_propen_push_do_test.yml

```diff
@@ -18,23 +34,18 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup Java
-        uses: actions/setup-java@v4
-        with:
-          java-version: 21
-          distribution: temurin
-          cache: maven
+        uses: ./.github/actions/setup-java
 
       - uses: graalvm/setup-graalvm@v1
         with:
           java-version: 21
           distribution: 'graalvm'
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
-      - run: ./mvnw --version
+      - name: Extract Maven Artifacts
+        uses: ./.github/actions/extract-maven-artifacts
 
-      - name: Maven build
-        run: >
-            ./mvnw clean install -s maven-settings.xml -B -e -DskipTests -Pnative
+      - run: ./mvnw --version
```

Comment:
is this for debugging? To print the versions, you could add `-V` to the `mvn verify`.

Reply:
I *think* so, this step already existed and I was trying to disturb the existing workflow as much as possible. I can remove and add `-V` to `mvn verify` though :+1: